katana/build.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 27
+
+    defaultConfig {
+        applicationId "com.mototdd.katana"
+        minSdkVersion 23
+        targetSdkVersion 27
+        versionCode 1
+        versionName "1.0"
+    }
+}
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
+    implementation 'com.android.support:appcompat-v7:27.1.0'
+
+    // https://github.com/mannodermaus/android-junit5
+    testImplementation 'junit:junit:4.12'
```

Comment:
is this for JUnit5?

Reply:
Ah, forgot to investigate this. Here, no, it is still 4.x. I will create a ticket for this upgrade. It does not appear as easy as I thought.

- [ ] Create jUnit5 migration ticket.